readthedocs_theme/templates/includes/topnav.html

```diff
@@ -67,6 +70,19 @@
   </a>
 {% endmacro %}
 
+{# Application links to sign up #}
+{% macro menu_sign_up() %}
+  <a class="item" data-analytics="community-signup" href="https://readthedocs.org/accounts/signup/">
+    <i class="fad fa-people-group primary icon"></i>
+    Read the Docs Community
+  </a>
+  <a class="item" data-analytics="commercial-login" href="https://readthedocs.com/accounts/signup/">
+    <i class="fad fa-building secondary icon"></i>
+    Read the Docs for Business
+  </a>
+{% endmacro %}
```

Comment:
Are we using this menu anywhere? Seems we can likely drop this entirely, in favor of the modal.